assembly/dependencies-apache-ignite.xml

```diff
@@ -205,6 +205,9 @@
         </fileSet>
         <fileSet>
             <directory>${basedir}/../control-center-agent/modules/control-center-agent/target/libs</directory>
+            <includes>
+                <include>commons-logging*.jar</include>
+            </includes>
```

Comment:
I don't think that we should filter agent 's libs in CE repository. Let's add exclusion to `copy-libs-dev` target in CCA repository. Like we do for org.apache.ignite,org.gridgain artifacts

Reply:
We could make `copy-libs-dev` copy only the `commons-logging` jar into the `libs` directory, but `copy-libs-dev` is executed at the test-compile stage. At that point the shading hasn't taken place yet. 
Am I missing something?

Reply:
yes, `copy-libs-dev` is executed at the test-compile stage. In `libs` directory you should copy deps that should be placed in module dir in release.

Reply:
Removed shading of commons-logging